components/brave_welcome_ui/components/welcome/index.tsx

```diff
@@ -71,34 +83,44 @@ function Welcome () {
 
   return (
     <S.Box ref={shouldPlayAnimations ? ref : null}>
-      <div className="view-logo-box">
+      <div className='view-logo-box'>
         <img src={braveLogoUrl} />
       </div>
-      <div className={classnames({ 'view-content': true, 'initial': shouldPlayAnimations })}>
-        <div className="view-header-box">
-          <div className="view-details">
-            <h1 className="view-title">{getLocale('braveWelcomeTitle')}</h1>
-            <p className="view-desc">{getLocale('braveWelcomeDesc')}</p>
+      <div
+        className={classnames({
+          'view-content': true,
+          'initial': shouldPlayAnimations
+        })}
+      >
+        <div className='view-header-box'>
+          <div className='view-details'>
+            <h1 className='view-title'>{getLocale('braveWelcomeTitle')}</h1>
+            <p className='view-desc'>{getLocale('braveWelcomeDesc')}</p>
```

Comment:
There are changes in this file related to Prettier rules that aren't directly related to the PR. We suggest applying those style changes across the directory in a separate, isolated PR. Could you please revert these changes.